public/js/invoice.js

```
@@ -0,0 +1,40 @@
+(function () {
+    'use strict';
+
+    const invoiceContainerSelector = '.invoice';
+    const invoiceContainer = document.querySelector(invoiceContainerSelector);
+
+    const invoiceId = localStorage.getItem('invoice');
+
+    const downloadInvoiceButton = document.createElement('button');
+    const invoiceTextParagraph = document.createElement('p');
+
+    const invoiceDownloadingHandler = () => {
+        getData(`http://localhost:3000/api/invoice/get?id=${invoiceId}`, 'blob')
+            .then((invoice) => {
+                window.open(window.URL.createObjectURL(invoice));
+            });
```

Missing `catch`/`done`. Errors will be lost.